settings/src/main/java/org/odk/collect/settings/ODKAppSettingsMigrator.java

```diff
@@ -124,11 +126,13 @@ private List<Migration> getUnprotectedMigrations() {
 
                 translateValue("never").toValue("every_fifteen_minutes").forKey("periodic_form_updates_check"),
 
-                moveKey("knownUrlList").toPreferences(metaPrefs)
+                moveKey("knownUrlList").toPreferences(metaSettings),
+
+                moveKey("default_completed").toPreferences(protectedSettings)
```

Comment:
I moved this key to protected settings first so that it's easy to perform the migrations (all keys are in the same settings). Otherwise, I would need to implement something like a cross-settings migration which would be more complex.